Libraries/Blob/File.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule File
+ * @flow
+ * @format
+ */
+'use strict';
+
+const Blob = require('Blob');
+
+const invariant = require('invariant');
```

Comment:
Based on other callsites in this repo, is this supposed to be `fbjs/lib/invariant`? https://github.com/facebook/react-native/search?utf8=%E2%9C%93&q=invariant&type=

Reply:
I made this a long time ago but it never got reviewed, could be useful https://github.com/facebook/react-native/pull/13014/files